src/browser/chat/ChatRoomPage.js

```diff
@@ -0,0 +1,100 @@
+/* @flow */
+import React, { Component } from 'react';
+import { PageHeader, Title, View, Input, Form } from '../app/components';
+import { connect } from 'react-redux';
+import Chat from './ChatPage';
+import { joinRoom, sendMessage, loadMessages, kickUserFromRoom } from '../../common/lib/redux-firebase/actions';
+
+const ChatRoomMessage = ({ sender, content }) => (
+  <View>
+    {sender}
+    : {content}
+  </View>
+);
+
+class ChatRoomMessages extends Component {
+  componentWillMount() {
+    this.props.loadMessages(this.props.chatRoom.id);
+  }
+  onInputKeyDown = (event) => {
```

Comment:
Prefer using an input type="submit" in the form to automatically the submit event when pressing Enter (this is better for accessibility and more standard overall).